clang/include/clang/Frontend/FrontendOptions.h

```diff
@@ -383,6 +383,9 @@ class FrontendOptions {
   LLVM_PREFERRED_TYPE(bool)
   unsigned ModulesShareFileManager : 1;
 
+  /// Connect to module build daemon
+  unsigned ModuleBuildDaemon : 1;
```

Comment:
Mark this with `LLVM_PREFERRED_TYPE(bool)`, like the other bool-like bitfield in this class.